io_scene_godot/converters/mesh.py

```diff
@@ -249,44 +289,35 @@ def export_morphs(self, export_settings, surfaces):
         )
         self.mesh_resource["blend_shape/mode"] = 0
 
-        # toggle shapekey uncompatible modifiers to false
-        modifier_config_cache = list()
-        if export_settings['use_mesh_modifiers']:
-            if not self.validate_morph_mesh_modifiers(
-                    self.object):
-                logging.warning(
-                    "Mesh object '%s' has modifiers "
-                    "incompatible with shape key",
-                    self.object.name
-                )
-
-            for modifier in self.object.modifiers:
-                modifier_config_cache.append(modifier.show_viewport)
-                modifier.show_viewport = False
-
-        # turn on shape key mode
-        self.object.show_only_shape_key = True
-        blender_shape_keys = self.object.data.shape_keys
-        for index, shape_key in enumerate(blender_shape_keys.key_blocks):
-            if shape_key == blender_shape_keys.reference_key:
+        # Ran some tests, subsurface modifier and other modifiers appear to
```

Comment:
I just did some test, yeah modifiers works fine with shape key now. Thanks for your effort! I think you can remove this part of code now.